tests/testthat/test_functions.R

```diff
@@ -74,6 +74,25 @@ test_that("matrix functions work as expected", {
 
 })
 
+test_that("kronecker works with greta and base array arguments", {
+    a <- rWishart(1, 6, diag(5))[, , 1]
```

Comment:
could you please add:

```r
  skip_if_not(check_tf_version())
  source("helpers.R")
```

This will skip these tests when run on a system that doesn't have TF and TFP installed (like CRAN) and also reset the RNG seed.

Reply:
Done.